ui-web/sdk/src/index.ts

```diff
@@ -17,14 +19,16 @@ export async function launchIframe(
   style.id = "trinsic-ui-style";
   style.textContent = CSSString;
   document.head.appendChild(style);
-  MicroModal.init();
-  if (launchUrl === undefined || launchUrl === null || launchUrl === "") {
+
+  if (!launchUrl) {
```

Comment:
FYI, since this runs everywhere, this one feels "unsafe" compared to what was. It will run in the browser etc, people can just throw into it what they want.

Reply:
@janpieterz I don't how the previous one is any better though. Making `!launchUrl` only really adds `=== NaN`, `=== false`, and `=== 0` to the condition. They could still throw anything really there. I'm not sure what's practically different here.